fluentd/generate_throttle_configs.rb

```diff
@@ -2,16 +2,35 @@
 require 'date'
 require 'logger'
 
+log_level = ENV['LOG_LEVEL'] || 'INFO'
```

Comment:
I set to INFO because of this log message https://github.com/openshift/origin-aggregated-logging/pull/350/files#diff-12cce37c2f5e5292a8adf8d11adae7b1R232 but maybe it makes sense to keep them the same.  I'm indifferent